modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -35,6 +35,11 @@ module LocaleModelHelpSetup {
 
   config param debugLocaleModel = false;
 
+  // We would really like a class-static storage class.(C++ nomenclature)
```

Comment:
I thought that existed? Or do we just have 'type methods' but not 'type variables' ?